test/powershell/Language/Interop/DotNet/DotNetInterop.Tests.ps1

```diff
@@ -1,7 +1,163 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
-Describe ".NET class interoperability" -Tags "CI" {
-    It "Should access types in System.Console" {
-        [System.Console]::TreatControlCAsInput | Should -BeFalse
+
+Describe "Handle ByRef-like types gracefully" -Tags "CI" {
+
+    BeforeAll {
+        $code = @'
+using System;
+namespace DotNetInterop
+{
+    public class Test
+    {
+        public static Span<int> Space
```

Comment:
Maybe also test an indexer.

Reply:
Oh, I forgot to handle `PSGetIndexerBinder`, will update.